alpha/apps/kaltura/lib/rules/conditions/kDeliveryProfileCondition.php

```diff
@@ -49,7 +49,7 @@ protected function internalFulfilled(kScope $scope)
 		}
 
 		KalturaLog::debug("Delivery profile ids [".print_r($profileIds, true)."]");
-		$requestOrigin = @$_SERVER['HTTP_X_FORWARDED_HOST'];
+		$requestOrigin = isset($_SERVER['HTTP_X_FORWARDED_HOST']) ? $_SERVER['HTTP_X_FORWARDED_HOST'] : null ;
```

Comment:
we dont need the @ anymore?

Reply:
No because I check if its set before accessing it